pkg/sql/logictest/testdata/logic_test/restore

```diff
@@ -0,0 +1,5 @@
+# LogicTest: default
+
+# Restore is CCL only.
+statement error pq: unknown statement type: \*tree\.Restore
+RESTORE foo FROM "bar"
```

Comment:
Can you add `BACKUP` just for completeness's sake? (And perhaps call this file "ccl" or "enterprise" or something more generic.)

Reply:
Done. renamed `ccl`.